aws/resource_aws_emr_instance_group.go

```diff
@@ -1,32 +1,73 @@
 package aws
 
 import (
+	"encoding/json"
 	"errors"
+	"fmt"
 	"log"
+	"strings"
 	"time"
 
-	"fmt"
-
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/emr"
 	"github.com/hashicorp/terraform/helper/resource"
 	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/structure"
+	"github.com/hashicorp/terraform/helper/validation"
 )
 
-var emrInstanceGroupNotFound = errors.New("No matching EMR Instance Group")
+var errEMRInstanceGroupNotFound = errors.New("No matching EMR Instance Group")
 
 func resourceAwsEMRInstanceGroup() *schema.Resource {
 	return &schema.Resource{
 		Create: resourceAwsEMRInstanceGroupCreate,
 		Read:   resourceAwsEMRInstanceGroupRead,
 		Update: resourceAwsEMRInstanceGroupUpdate,
 		Delete: resourceAwsEMRInstanceGroupDelete,
+		Importer: &schema.ResourceImporter{
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				idParts := strings.Split(d.Id(), "/")
+				if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" {
+					return nil, fmt.Errorf("Unexpected format of ID (%q), expected cluster-id/ig-id", d.Id())
+				}
+				clusterID := idParts[0]
+				resourceID := idParts[1]
+				d.Set("cluster_id", clusterID)
+				d.SetId(resourceID)
+				return []*schema.ResourceData{d}, nil
+			},
+		},
 		Schema: map[string]*schema.Schema{
+			"autoscaling_policy": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				DiffSuppressFunc: suppressEquivalentJsonDiffs,
+				ValidateFunc:     validation.ValidateJsonString,
+				StateFunc: func(v interface{}) string {
+					s, _ := structure.NormalizeJsonString(v.(string))
+					return s
+				},
+			},
+			"bid_price": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+			},
 			"cluster_id": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
+			"instance_role": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Default:  "TASK",
```

Comment:
Nit: We should prefer to use the [SDK provided constants](https://docs.aws.amazon.com/sdk-for-go/api/service/emr/#pkg-constants) in the `Default` here and `ValidateFunc` below, e.g.

```suggestion
				Default:  emr.InstanceGroupTypeTask,
```